src/store/selectors/index.js

```diff
@@ -31,7 +31,6 @@ export const allCaptured = createSelector(
 
 export const captureSelector = createSelector(
   validCaptures,
-  validCapturesValue => mapKeys(
```

Comment:
this selector was being used publicly and internally. It was simplified for internal use and the public api was maintained, but the logic was moved to events.js